extensions/blocks/podcast-player/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Returns a class based on the context a color is being used and its slug.
```

Comment:
Might be worth mentioning why we are not using the one from Gutenberg Core here just in case.

Reply:
> > What do you think about kicking it to another next one?
> 
> Sure thing.

Done some fixes [here](https://github.com/Automattic/jetpack/pull/15110/commits/a26dc10a3b1cc6788638d438da76e6e79a333061).